docs/wallets/decrediton/decrediton-setup.md

```diff
@@ -8,7 +8,7 @@ Last updated for Decrediton v{{ decreditonversion }}.
 
 !!! info "Blockchain Download"
 
-    Without [SPV](../../wallets/spv.md) enabled, Decrediton needs to download a full copy of the Decred blockchain before it can be used. This means that a large download must be completed and a large amount of storage space must be available on the device where Decrediton is installed. You can check the current size of all serialized blocks via [dcrdata charts](https://explorer.dcrdata.org/charts#blockchain-size). The total size of Decrediton will be bigger, since additional wallet data is stored on your device.
+    Without [SPV](../../wallets/spv.md) enabled, Decrediton needs to download a full copy of the Decred blockchain before it can be used. This means that a large download must be completed and a large amount of storage space must be available on the device where Decrediton is installed. You can check the current size of all serialized blocks via [dcrdata charts](https://dcrdata.decred.org/charts#blockchain-size). The total size of Decrediton will be bigger, since additional wallet data is stored on your device.
```

Comment:
This link does not open the correct chart

Reply:
Looks like the correct URL should be https://dcrdata.decred.org/charts?chart=blockchain-size